src/inc/ex.h

```diff
@@ -9,6 +9,20 @@
 
 void RetailAssertIfExpectedClean();             // Defined in src/utilcode/debug.cpp
 
+#ifdef FEATURE_PAL
+inline EXCEPTION_DISPOSITION __defaultExceptionFilter(PAL_SEHException& ex)
+{
+    return EXCEPTION_EXECUTE_HANDLER;
+}
+
+#define EX_TRY_HOLDER \
+    auto __exceptionHolder = NativeExceptionHolderFactory::CreateHolder(&__defaultExceptionFilter); \
+    __exceptionHolder.Push();
+
```

Comment:
This can be simplified to use another non-templated derived class of NativeExceptionHolderBase that overrides InvokeFilter to return EXCEPTION_EXECUTE_HANDLER.

class NativeExceptionHolderCatchAll : public NativeExceptionHolderBase
{

public:
    NativeExceptionHolderCatchAll()
        : NativeExceptionHolderBase()
    {
        Push();
    }

    virtual EXCEPTION_DISPOSITION InvokeFilter(PAL_SEHException& ex)
    {
        return EXCEPTION_EXECUTE_HANDLER;
    }
};

So then the EX_TRY_HOLDER can be just:
    auto NativeExceptionHolderCatchAll __exceptionHolder;